webapp/channels/src/components/dropdown_input.tsx

```diff
@@ -101,17 +95,37 @@ const DropdownInput = <T extends ValueType>(props: Props<T>) => {
         }
     };
 
+    const {formatMessage} = useIntl();
+    const [customInputLabel, setCustomInputLabel] = useState<CustomMessageInputType>(null);
+    const [ownValue, setOwnValue] = useState<T>();
```

Comment:
It would seem that `ownValue` is only used in callbacks, so could be set as a reference.

Reply:
Done.